syntax = "proto3";

service Agent {
    rpc GetDiskInfo(Request) returns (Response) {}
    rpc WriteCephConf(Request) returns (Response) {}
    rpc InstallPackage(Request) returns (Response) {}
    rpc StartService(Request) returns (Response) {}
}

message Request {
  string key = 1;
}

message Response {
  string value = 1;
}
